content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md

```diff
@@ -0,0 +1,379 @@
+---
+db_id: 
+available_flavours:
+- kotlin
+content_type: project
+prerequisites:
+  hard:
+  - projects/kotlin/project-5/live-data-observers
+  soft: []
+ready: true
+submission_type: repo
```

Comment:
do you mean `continue_repo` or `repo`?  
Should this project be on the same code base as `projects/kotlin/project-5/live-data-observers`?

Reply:
Always separate.